components/alert/no-js-alerts/no-js-alert.html

```diff
@@ -20,8 +20,11 @@
         <header class="et_landing-page-header main-page-header">
             <div class="et_landing-page-header-inner-container">
                 <!-- header-inner-container child1 starts -->
-                <div class="et_inner-container-child1"><img src="/Images/Expeditech UI-logos.jpeg" alt="Expeditech UI"
-                        width="130" class="et_logo" /></div>
+                <div class="et_inner-container-child1">
+                    <a href="/">
+                        <img src="/Images/Expeditech-UI-logos.png" alt="Expeditech UI" width="130" class="et_logo" />
+                    </a>
```

Comment:
Add rel="noopener" or rel="noreferrer" to any external links to improve performance and prevent security vulnerabilities.

chech this link : https://stackoverflow.com/questions/50709625/link-with-target-blank-and-rel-noopener-noreferrer-still-vulnerable

Reply:
didn't know about this, thankyou Mehul for pointing and suggesting it out!!